templates/eth1tx.html

```diff
@@ -47,14 +112,23 @@ <h1 class="h4 mb-1 mb-md-0">
     {{ with .Data }}
       <div class="card">
         <div class="card-header">
+          <div id="advanced-itx" class="hide mr-20 float-right">
+            <div class="float-right">
+              <span class="text-monospace mr-2">Advanced</span>
+              <label class="advanced-itx-switch float-right" for="advanced-itx-toggle">
```

Comment:
**question:** Unlike Etherscan the `advanced-itx-switch` is visible and works in both the **Overview** and **Internal Transactions** tabs.
When clicked in the **Overview** tab you only see the number in the **Internal Transactions** tab change.

Should this be how it works?

Reply:
Ah no, this was not intended. I assume you're using firefox, because on chrome it works for me; I adopted the same fix we're using for the tabBarSwitcher, this should fix it. The list of internal transactions in the overview tab should only show non-advanced subcalls at all times.

Reply:
Yes, I am normally using Firefox.
It looks fixed for both Firefox and Chrome now.